main.cpp

```diff
@@ -296,7 +293,10 @@ int Bird::FlyingBird::flapsPerTrip(int minutesPerTrip)
 {
     while(minutesPerTrip > 0)
     {
-        return flapsPerMinute * minutesPerTrip;
+        int result;
+        result = flapsPerMinute * minutesPerTrip; 
+        --minutesPerTrip;
+        return result;       
```

Comment:
This loop will happen exactly once because you're returning the first time it runs.
why even have the loop if you're doing that?
- [ ] revise.